src/ui.js

```
@@ -96,6 +159,90 @@ function getPrimaryCardSelections() {
+
+function refreshVisibleTimerEntityFlag() {
+  hasVisibleTimerEntities = Array.from(visibleEntityIds).some((entityId) => (
+    isTimerEntityForLiveUpdates(state.STATES?.[entityId])
+  ));
+}
+
+function refreshVisibleEntityCache() {
+  try {
+    const nextVisibleIds = new Set();
+    const favorites = (state.CONFIG?.favoriteEntities || []).slice(0, 12);
+    favorites.forEach((entityId) => addVisibleEntityCandidate(nextVisibleIds, entityId));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)